draft-yasskin-dispatch-web-packaging.md

```diff
@@ -0,0 +1,650 @@
+---
+coding: utf-8
+
+title: Web Packaging
+docname: draft-yasskin-dispatch-web-packaging-latest
+category: std
+
+ipr: trust200902
+area: gen
+workgroup: Dispatch
+keyword: Internet-Draft
+
+stand_alone: yes
+pi: [comments, sortrefs, strict, symrefs, toc]
+
+author:
+ -
+    name: Jeffrey Yasskin
+    organization: Google
+    email: jyasskin@chromium.org
```

Comment:
Folks seem to vary. Several people use @google, but https://tools.ietf.org/html/rfc6797 uses a personal address.